sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -156,7 +156,9 @@ export module ConnectionContextBase {
         platform: `(${os.arch()}-${os.type()}-${os.release()})`,
         framework: `Node/${process.version}`
       },
-      operationTimeoutInSeconds: parameters.operationTimeoutInSeconds
+      operationTimeoutInSeconds: parameters.operationTimeoutInMs
+        ? parameters.operationTimeoutInMs / 1000
+        : undefined
```

Comment:
Should we use some default timeout here?

Reply:
Ahh... If the timeout in undefined, we are setting `defaultOperationTimeoutInSeconds` i.e 60 seconds in rhea-promise

Reply:
Yeah, since we were previously not setting any defaults here, not doing it
(because purpose is just to address naming and unit conversion!)